CHANGES.rst

```diff
@@ -6,6 +6,7 @@
 - [FIXED] Fixed Cloudant exception code 409 with 412 when creating a database that already exists.
 - [FIXED] Catch error if ``throw_on_exists`` flag is ``False`` for document create.
 - [FIXED] Fixed /_all_docs call where ``keys`` is an empty list.
+- [FIXED] Use unicode Null U+0000 as /_all_docs iterator ``startkey`` parameter.
```

Comment:
This is how you fixed it, but it would be better for users to know what problem was fixed. Something like:
>[FIXED] Issue where docs with IDs that sorted lower than 0 were not returned when iterating through _all_docs